app/src/main/java/com/duckduckgo/app/pixels/AppPixelName.kt

```diff
@@ -225,6 +225,8 @@ enum class AppPixelName(override val pixelName: String) : Pixel.PixelName {
     FIRE_ANIMATION_NEW_SELECTED("m_fas_s"),
 
     EMAIL_TOOLTIP_DISMISSED("email_tooltip_dismissed"),
+    EMAIL_ENABLED("email_enabled"),
```

Comment:
I've skipped the `m_` prefix assuming is ok for the email pixels. Let me know if that's ok.